src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -486,23 +429,17 @@ public void updateSubscribers()
         Set<EntityPlayerMP> oldSubscribers = subscribers;
         subscribers = new HashSet<>();
 
-        //  Add owners
-        for (Object o : MinecraftServer.getServer().getConfigurationManager().playerEntityList)
-        {
-            if (o instanceof EntityPlayerMP)
-            {
-                EntityPlayerMP player = (EntityPlayerMP)o;
-                if (permissions.isSubscriber(player))
-                {
-                    subscribers.add(player);
-                }
-            }
-        }
+        // Add owners
+        subscribers.addAll(
+                MinecraftServer.getServer().getConfigurationManager().playerEntityList
+                        .stream()
+                        .filter(permissions::isSubscriber)
+                        .collect(Collectors.toList()));
 
         //  Add nearby players
         if (world != null)
         {
-            for (Object o : world.playerEntities)
+            for (EntityPlayer o : world.playerEntities)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'o' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)